src/components/PopupTrigger.js

```diff
@@ -20,27 +22,35 @@ const styles = {
 
 // This is written as a "function" function rather than an arrow function because react-onclickoutside wants it to have a prototype
 // eslint-disable-next-line prefer-arrow-callback
-export const Popup = onClickOutside(function({ side = 'right', target: targetId, onClick, children }) {
+export const Popup = onClickOutside(function({ id, side = 'right', target: targetId, onClick, children, popupProps = {} }) {
+  // We're passing popupProps here rather than just props, because ...props also includes lots of internal onClickOutside properties which
+  // aren't valid to be dropped on a DOM element.
+  Utils.useConsoleAssert('aria-label' in popupProps || 'aria-labelledby' in popupProps, 'In order to be accessible, Popup needs a label')
```

Comment:
We don't tend to use `in`, so to match other places:
```suggestion
  Utils.useConsoleAssert(popupProps['aria-label'] ||  popupProps['aria-labelledby'], 'In order to be accessible, Popup needs a label')
```

Reply:
Ok, sounds good. I think I had been trying to account for someone intentionally setting the `aria-label` to `''`, but that should probably also throw a warning. I'll clean all of these useConsoleAssert statements in a new PR anyway.